### ISSUE TITLE
Scrolling behavior for stopping a scroll does not feel native

### ISSUE THREAD

**Issue by georgbachmann** [mapbox/mapbox-maps-ios#645]
## Environment

- Xcode version: Xcode 12.5
- iOS version: iOS 14.7.1
- Maps SDK Version: 1.0-RC7

## Observed behavior and steps to reproduce

When looking at e.g the AppleMaps app or Google Maps app or even the old Mapbox-Maps (is it v6?!? not sure...)
So you scroll inside the map and then fling (is that the word? (I an not native english) but I mean "rapidly move") your finger across the map and let go of the finger, then the scrollview (MapView) will then keep moving and start decelerating. That is also the case in the new V10.

But on a regular UIScrollview (and therefore most likely the mentioned map renderers above) stop this deceleration when the user buts down a finger.

The new V10 mapview does not do that only by putting down the finger. It does stop when I but down my finger while this deceleration is in progress AND move my finger a but, but not only by putting it down.

## Expected behavior

The map scroll stops immediately when I put down my finger while it moves

I hope I was able to describe the problem?!? If not please tell me and I'll try to clarify.

**Comment by macdrevx** [comment]
Thanks for opening this issue. It does make sense.

**Comment by arthurschiller** [comment]
I was about to open the same ticket.
The deceleration behavior feels odd and kinda off. Related to this setting the mapView.gestures.decelerationRate property to for example UIScrollView.DecelerationRate.normal.rawValue does not seem to make any difference compared to the default value.

**Comment by macdrevx** [comment]
@arthurschiller, you're right. There are some some known issues here, and we plan to make improvements in this area.

**Comment by georgbachmann** [comment]
@macdrevx I just tried out the latest release 10.6.0 and to me it still does not feel as it should?!?

I move my map with a rapid fling of my finger (which i usually do to get around the map quickly) and then put a finger down where I want to stop.

First: The map does not immediately stop, but continues to move a bit further.

Second: Now the pan I do after putting down my finger is ignored. So fling - finger down to stop - fling again (cause I am still not far enough)... does not work.

Third: Subsequent pans seem broken. I e.g. can't fling the map afterwards. It will not really move. Not sure how to describe it, but it feels off. It also triggers the double-tap-and-pan-to-zoom gesture quite often.


I still think that the way the map behaved in the old (pre 10) version is much much closer to what it should be (compared to Apple Maps and Google Maps) and it's still holding me back to update to v10 cause the map is a very essential part in our app and that needs to feel spot on.

Would be awesome if you could have an other look at that issue please :)

**Comment by macdrevx** [comment]
@ZiZasaurus @evil159

**Comment by ZiZasaurus** [comment]
@georgbachmann can you please provide the following information so we can dig a bit deeper:
1. What device(s) are you seeing this behavior on?
2. Can you use [this link ](https://support.apple.com/en-us/HT208721) to provide us with a video reproduction of the behaviors you're experiencing? If you're using a physical device, are you able to take a video of yourself interacting with the map and reproducing these behaviors? Either of these will allow us to see your gestures on screen and pinpoint the root cause.
